config/routes.rb

```diff
@@ -8,5 +16,9 @@
   resources :destinations, only: [:index, :show]
 
   get '/cart', to: 'cart#show'
+  get '/login', to: "sessions#new"
+  post '/login', to: "sessions#create"
+  delete '/logout', to: "sessions#destroy", as: "logout"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.